rules/Php80/Rector/NotIdentical/StrContainsRector.php

```diff
@@ -100,7 +100,7 @@ public function refactor(Node $node): ?Node
         if (isset($funcCall->getArgs()[2])) {
             $secondArg = $funcCall->getArgs()[2];
 
-            if ($this->isName($funcCall->name, 'strpos') && ! $this->isIntegerZero($secondArg->value)) {
+            if ($this->isNames($funcCall->name, ['strpos', 'mb_strpos']) && ! $this->isIntegerZero($secondArg->value)) {
```

Comment:
test is needed for substr with multibyte usage

Reply:
see this test

https://github.com/rectorphp/rector-src/blob/8e6009063331a3656fa5f77b28f5779dba71c6d6/rules-tests/Php80/Rector/NotIdentical/StrContainsRector/Fixture/offset_variable_equal_strpos.php.inc#L5-L26

the multibyte verification is needed for mb_strpos.

Reply:
Added the same test for `mb_strpos`